nemo/collections/cv/modules/video_augment.py

```diff
@@ -0,0 +1,185 @@
+import random
+
+import torch
+import torchaudio
+import torchvision
```

Comment:
Torchvision is not a direct dependency of NeMo and is very difficult to install in container. Please make it optional dependency with try catch check like for numba. 

Inside init, test if the torchaudio library is available and if user has requested this augmentation. If available, use it, if not, raise appropriate warning that torchaudio is not installed and so this augmentation cannot be used.

Reply:
Also, can you check if torchvision is pre-installed in the nvidia containers? In that case at least it should be fine in container. For pip conda users we can ask them to manually install torchvision locally

Reply:
It's available in our containers (and I think on the pytorch website, they always show installation of torch with torchvision included https://pytorch.org/get-started/locally/)

Reply:
Still, all Torch* libraries must be guarded. It's a pain to build release containers when the container version fails or if users pip install just torch instead of conda

Reply:
done, I added try/catch in cv/modules/video_aument.py, cv/modules/video_preprocessing.py and cv/parts/preprocessing/features.py.